agent/util-scripts/gold/pbench-clear-tools/test-12.txt

```diff
@@ -1,7 +1,7 @@
 +++ Running test-12 pbench-clear-tools 
-All tools removed from host, "fubar"
-Removed "mpstat" from host, "testhost.example.com", in tools group, "default"
-All tools removed from host, "testhost.example.com"
+[36mAll tools removed from host, "fubar"[0m
```

Comment:
Is there a way for color logs to detect when the console is an actual terminal?

Reply:
Fixed, colorama needed to be installed

Reply:
Seems like `colorama` does not make a difference, as the coloring escape sequences are present still.